docs/api.rst

```diff
@@ -93,9 +93,12 @@ The KSQL resource runs a sequence of KSQL statements. All statements, except tho
    :>json string  sourceDescription.name: The name of the stream or table.
    :>json array   sourceDescription.readQueries: The queries reading from the stream or table.
    :>json array   sourceDescription.writeQueries: The queries writing into the stream or table
-   :>json array   sourceDescription.schema: The schema of the stream or table as a list of column names and types.
-   :>json string  sourceDescription.schema[i].name: The name of the column.
-   :>json string  sourceDescription.schema[i].type: The data type of the column.
+   :>json array   sourceDescription.fields: The schema of the stream or table as a list of column names and types.
+   :>json string  sourceDescription.fields[i].name: The name of the column.
+   :>json string  sourceDescription.fields[i].schema: The schema of the column.
+   :>json string  sourceDescription.fields[i].schema.type: The type the schema represents. One of INTEGER, BIGINT, BOOLEAN, DOUBLE, STRING, MAP, ARRAY, or STRUCT.
+   :>json string  sourceDescription.fields[i].schema.memberSchema: For MAP and ARRAY types, contains the schema of the map values and array elements, respectively. For other types this field is not used and its value is undefined.
+   :>json string  sourceDescription.fields[i].schema.fields: For STRUCT types, contains a list of the fields within the struct.
```

Comment:
why is `sourceDescription.fields[i].schema.fields` a string? It says here it's a list.

Reply:
That's an error